.github/workflows/check-file-updates.yaml

```diff
@@ -0,0 +1,43 @@
+name: Check config and readme updates
+on:
+  pull_request:
+    paths:
+      - "apis/**"
+jobs:
+  file-updates:
+    permissions:
+      pull-requests: write
+    name: Ensure generated files are included
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Generate files
+        run: |
+          make generate manifests api-docs
+      - name: Ensure generated files are up-to-date
+        id: check_generated_files
+        run : |
+          if [[ -n $(git status -s) ]]
+          then
+            echo "Generated files have been missed in the PR"
+            echo "missing_generated_files=true" >> $GITHUB_OUTPUT
+          else
+            echo "No new files to commit"
+            echo "missing_generated_files=false" >> $GITHUB_OUTPUT
+          fi
+      - name: Report issue in PR
+        if: ${{ steps.check_generated_files.outputs.missing_generated_files == 'true' }}
+        uses: thollander/actions-comment-pull-request@v2
+        with:
+          message: |
+              ## This PR can't be merged just yet 😢
+
+              Hey there! It looks like some files aren't up to date. No worries though! Just run the commands below to automatically generate those files and make sure to commit them afterwards:
+                - `make generate manifests api-docs`
+            
+              For more info: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
```

Comment:
aint this already in the standard test comments block or still want to detail here?

Reply:
i think it will be ended as a new line in this block, thats where we normally find "Details" of failure test, isnt it? 
![Screenshot from 2024-03-14 14-26-45](https://github.com/opendatahub-io/opendatahub-operator/assets/915053/bfb766f0-ee00-42ef-89d8-7a5b5d09f7e5)
but i am not against to have it posted in the comments,

Reply:
hmm initially, I thought that as soon as a user sees a gh action comment, he can have access to the failed job easily.
Just reducing the number of clicks :P

Reply:
ok, lets keep it then